packages/contracts/contracts/Poll.sol

```diff
@@ -182,10 +182,7 @@ contract Poll is Params, Utilities, SnarkCommon, IPoll {
   /// @dev Can only be submitted before the voting deadline
   /// @param _messages the messages
   /// @param _encPubKeys the encrypted public keys
-  function publishMessageBatch(
-    Message[] calldata _messages,
-    PubKey[] calldata _encPubKeys
-  ) public virtual isWithinVotingDeadline {
+  function publishMessageBatch(Message[] calldata _messages, PubKey[] calldata _encPubKeys) public virtual {
```

Comment:
because this function calls publishMessage in a loop, and publishMessage already has isWithinVotingDeadline so it's not necessary to double check (you can check the tests to confirm)

Reply:
thanks!